src/lib/seam/components/ClimateSettingScheduleDetails/ClimateSettingScheduleDetails.stories.tsx

```diff
@@ -0,0 +1,57 @@
+import { Button, Dialog } from '@mui/material'
+import type { Meta, StoryObj } from '@storybook/react'
+
+import { ClimateSettingScheduleDetails } from 'lib/seam/components/ClimateSettingScheduleDetails/ClimateSettingScheduleDetails.js'
+import { useToggle } from 'lib/ui/use-toggle.js'
+
+/**
+ * These stories showcase access code details.
+ */
+const meta: Meta<typeof ClimateSettingScheduleDetails> = {
+  title: 'Example/ClimateSettingScheduleDetails (Coming Soon)',
+  component: ClimateSettingScheduleDetails,
+  tags: ['autodocs'],
+  parameters: {
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/file/WTdzEh4CudGdQFOtuVR3Eh/Thermostat%2FNoise-components?type=design&node-id=830-88257&mode=design&t=PEFHm2uapLILRbqf-0',
+    },
+  },
+}
+
+export default meta
+
+type Story = StoryObj<typeof ClimateSettingScheduleDetails>
+
+export const Content: Story = {
+  render: (props) => (
+    <ClimateSettingScheduleDetails
+      {...props}
+      climateSettingScheduleId={
+        props.climateSettingScheduleId ?? 'climate_setting_schedule1'
+      }
+    />
+  ),
+}
+
+export const InsideModal: Story = {
+  render: (props) => {
+    // eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
I think we disabled this for all stories globally—could remove

Reply:
ref https://github.com/seamapi/react/issues/366

Reply:
[6e7d9a3](https://github.com/seamapi/react/pull/375/commits/6e7d9a3e75e866d750fbe66b1c2e0fa3a2ac5bea)